gate-core/src/main/java/com/netflix/spinnaker/gate/services/internal/ClouddriverService.java

```diff
@@ -338,6 +338,10 @@ Response getAdhocData(
   @GET("/ecs/secrets")
   List<Map> getAllEcsSecrets();
 
+  @GET("/ecs/ecsDescribeClusters/{account}/{region}")
+  List<Map> getAllEcsClustersDescription(
```

Comment:
similar to [this comment](https://github.com/spinnaker/clouddriver/pull/5149/files#r539549768) on a related PR, we should keep the path and controller method names consistent (here and in clouddriver). FWIW I think _ecsClusterDescription**s**_ is the right noun to use in all cases.